app/models/reaction.rb

```diff
@@ -13,7 +13,7 @@ class Reaction < ApplicationRecord
   counter_culture :user
 
   validates :category, inclusion: { in: CATEGORIES }
-  validates :reactable_type, inclusion: { in: %w[Comment Article] }
+  validates :reactable_type, inclusion: { in: %w[Comment Article User] }
```

Comment:
how would you feel at moving `%w[Comment Article User]` to a constant? Kind of like we're doing in the previous line with `CATEGORIES`

Reply:
👍 makes sense!